contracts/strategies/algebra/AlgebraConverterStrategyLogicLib.sol

```diff
@@ -24,8 +24,12 @@ library AlgebraConverterStrategyLogicLib {
 
   //region ------------------------------------------------ Events
   event Rebalanced(uint loss, uint profitToCover, uint coveredByRewards);
+  event RebalancedDebt(uint loss, uint profitToCover, uint coveredByRewards);
```

Comment:
New event is added, it worth to add parsing of this event to CaptureEvents.ts

Reply:
Parsing added, not used in ci tests now. This emited data used and tested in backtester.